diagrams-svg.cabal

```diff
@@ -61,6 +62,7 @@ Library
                      , lens                 >= 4.0   && < 4.12
                      , hashable             >= 1.1   && < 1.3
                      , optparse-applicative >= 0.10  && < 0.12
+                     , semigroups
```

Comment:
Could you add version bounds for `semigroups`?  From a quick look at [the changelog](http://hackage.haskell.org/package/semigroups-0.16.2.2/changelog) it looks to me like `>= 0.13 && < 0.17` ought to work.